sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc

```diff
@@ -117,52 +143,56 @@ The base pointer determines the starting address of the matrix to be loaded/stor
 	
-IMPORTANT: In the current implementation, the layout in the load of matrix B must be `packed_b`.  Therefore, both the template parameter for the declaration of the B matrix and the call to `joint_matrix_load` for the B matrix must specify the `packed_b` layout.  The layout in the load of matrices A and C must be `row_major`, and the layout in the store of matrix C must also be `row_major`.
+IMPORTANT: In the current AMX and DPAS implementation, the layout in the load of matrix B must be `packed_b`.  Therefore, both the template parameter for the declaration of the B matrix and the call to `joint_matrix_load` for the B matrix must specify the `packed_b` layout.  The layout in the load of matrices A and C must be `row_major`, and the layout in the store of matrix C must also be `row_major`.
 
 Since the matrix functions are group operations (as defined in Section 4.17.3 of the SYCL specification), the matrix API has to be accessed by all the work-items in the group in a convergent control flow. The `Group` template argument can be a work-group or a subgroup. These functions will be called once by each work item in the group.
 
+**_NOTE:_** _The CUDA backend does not require any other sub-group size than 32, which is the size of a warp which acts as the sub-group, and also the default DPC++ sub-group size for the CUDA backend._
+
 To be aligned with the SYCL 2020 group algorithms, an additional group argument is added to the matrix operations to designate that these functions are collective operations. The {dpcpp} syntax is the following: 
 
 IMPORTANT: In the current implementation, only the subgroup scope is supported.  
 
 #### Load 
 ```c++
 namespace sycl::ext::oneapi::experimental::matrix {
-  template <typename Group, typename T, size_t NumRows, size_t NumCols,
-          matrix_layout Layout,
-          access::address_space Space>
-  void joint_matrix_load(Group sg, joint_matrix<T, NumRows, NumCols, Layout, Group> &res,
-		    multi_ptr<T, Space> src, size_t stride, matrix_layout MemLayout);
+template <typename Group, typename T, matrix_use MT, size_t NumRows,
+          size_t NumCols, matrix_layout Layout, access::address_space Space>
+void joint_matrix_load(
+    Group sg, joint_matrix<T, MT, NumRows, NumCols, Layout, Group> &res,
+    multi_ptr<T, Space> src, size_t stride);
 }
 ```
-This function loads data from memory to the 2d tiles/registers of Intel AMX/DPAS.
-
+This function loads data from memory to the 2d tiles/registers of Intel AMX/DPAS or to the matrix "fragments" for Nvidia Tensorcore.  Note that `Layout` is not included as an argument since it may be determined from the joint_matrix argument.
```

Comment:
Is it also acceptable for the other backends to determine matrix layout from joint_matrix instead of having an extra argument to joint_matrix_load/joint_matrix_store for `matrix_layout Layout`?

Reply:
+This function loads data from memory to the 2d tiles/registers of Intel AMX/DPAS/Nvidia tensorcores 

fragment is not the hardware name right? can you just refer to them as registers?
If not, you can still merge things as something like this:
+This function loads data from memory to the 2d tiles/registers/fragments of Intel AMX/DPAS/Nvidia tensorcores, respectively. 

The comment about "
 Note that `Layout` is not included as an argument since it may be determined from the joint_matrix argument."
should be added in the tensorcores subsection that adds use argument.